frontend/src/metabase/visualizations/visualizations/Heading/Heading.unit.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+import { Heading } from "../Heading";
+
+interface Settings {
+  text: string;
+}
+
+const defaultProps = {
+  onUpdateVisualizationSettings: ({ text }: { text: string }) => {
+    return;
+  },
+  dashcard: {},
```

Comment:
To simplify writing specs we have mock object factories like `createMockDashboardCardWithVirtualCard` in `frontend/src/metabase-types/api/mocks/dashboard.ts` — you will need it once you update the props types